app/src/main/java/com/android/sample/ui/mainscreen/MainScreen.kt

```diff
@@ -25,12 +25,14 @@ fun MainScreen(navigationActions: NavigationActions) {
         TopAppBar(
             title = { Text("Main Screen") },
             actions = {
-              IconButton(onClick = { navigationActions.navigateTo("Settings Screen") }) {
-                Icon(
-                    imageVector = Icons.Outlined.Settings,
-                    contentDescription = "Settings",
-                    tint = Color.Black)
-              }
+              IconButton(
```

Comment:
Again the code I added on MainScreen is similar but you might want to take a look at it.